pandas/tests/arrays/string_/test_string.py

```diff
@@ -431,23 +431,25 @@ def test_arrow_array(dtype):
 
 
 @td.skip_if_no("pyarrow")
-def test_arrow_roundtrip(dtype):
+def test_arrow_roundtrip(dtype, string_storage2):
```

Comment:
why aren't you just using `string_storage` (e.g. when you only need 1)?

Reply:
string_storage parameterizes dtype, so string_storage is the same as dtype and can't test the StringArray against pyarrow global storage setting and vice-versa.

this way we have 4 tests, not 2